docs/DEPLOYMENT_OVERVIEW.md

```diff
@@ -83,6 +83,10 @@ A service deployment lives in a GCP project configured with the items below.
     }
    ]
    ```
+    - The files stored in this bucket are either exported to VWB or downloaded to the user's computer, 
```

Comment:
The files being exported is not really part of Tanagra. We just write the files to the bucket, generate a signed URL, and either display that in the UI for the user to click on to download, or redirect to another configurable URL. So I think this statement is misleading. There is no VWB-specific code in Tanagra.

Reply:
How about this update